site/docs/v1/tech/apis/event-logs.adoc

```diff
@@ -103,10 +103,10 @@ The database column to order the search results on plus the order direction.
 +
 The possible values are:
 +
-* `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Event Log was created
-* `insertUser` - the user that create the Event Log
-* `message` - the message of the Event Log
-* `type` - the type of the Event Log
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Event Log was created
```

Comment:
I don't mind this. :-) I like that they are fixing layout issues.

But in general Dan makes a good point that it is easier to digest this as a separate PR since it is unrelated. I don't have a strong opinion either way.

Reply:
It's mildly related. Mark corrected them on the other search API documentation.